README.md

```diff
@@ -85,6 +87,8 @@ dialog.setContent("<p>Here's some new content!</p>");
 | dialog:destroyed | this | Fired when destroy() is called.
 | dialog:locked | this | Fired when lock() is called.
 | dialog:unlocked | this | Fired when unlock() is called.
+| dialog:hideclose | this | Fired when hideclose() is called.
+| dialog:showclose | this | Fired when showclose() is called.

```

Comment:
It looks like these aren't the actual events that are fired - it looks like `dialog:hiddenclose` and `dialog:showingclose` are actually thrown - this documentation is not accurate. I would recommend renaming the events to `dialog:closehidden` and `dialog:closeshown` in the documentation and the functions, it is more consistent.